tests/integration/olp-cpp-sdk-authentication/TokenEndpointTest.cpp

```diff
@@ -17,7 +17,7 @@
  * License-Filename: LICENSE
  */
 
-#include <olp/authentication/TokenEndpoint.h>
+#include "TokenEndpoint.h"
```

Comment:
Move down to the list of project includes.
The rule of own header first is only for own header and this is not the header of TokenEndpointTest.cpp

Reply:
got it